style.css

```diff
@@ -0,0 +1,65 @@
+
+@media only screen and (max-width: 600px) {
+  @font-face {
+    font-family: Poppins-Regular;
+    src: url(./Poppins.ttf);
+  }
+
+  body {
+    margin: 0;
+    background-image: url("./assets/bg\ image.png.png");
+  }
+
+  .first-section {
+    display: flex;
+  }
+
+  header {
+    background-color: rgb(239, 244, 249);
+  }
+
+  a {
+    text-decoration: none;
+    color: black;
+  }
+
+  nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-left: 20px;
+    margin-right: 20px;
+  }
+
+  .h1 {
+    font-family: Poppins-Regular, sans-serif;
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 52px;
+  }
+
+  p {
+    font-family: Poppins-Regular, sans-serif;
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  h3 {
+    color: rgb(0, 170, 255);
+    font-family: Poppins-Regular, sans-serif;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+  }
```

Comment:
- [ ] for best CSS practices  If you want to create a CSS rule for multiple elements, use a class attribute instead. For example, instead of using `font-family: Poppins-Regular, sans-serif;` 3 times you can create a class and add it to all elements that need to be popping family.



 ```CSS
 .poppins {font-family: Poppins-Regular, sans-serif;}
```